### DISCUSSION TITLE
How to install 1.5.2 ?

### DISCUSSION THREAD

**Discussion by mennax** [blakej115/magic-forms#12] [General]
Thanks for continuing @skydiver's much appreciated  work!
I try to apply version 1.5.2 in my October CMS version: v2.1.29 
but I recieve the error: Version file /Users/mennax/Sites/vicinus2/plugins/blakejones/magicforms/updates/added dutch translation [thanks to rubenvanerk].php is not found.
How to install 1.5.2 version as suggested?

**Comment by blakej115** [comment]
I was able to get the latest version of Magic Forms, v1.6.5, working with the latest version of October v2, October version v2.2.35. I've updated the plugin page to reflect this compatibility. If you are able to update to the latest version of October v2 or v3, I would recommend that.

Older versions of October v2 are unlikely to receive additional support. If you are version locked to October v2.1.29 for some reason, I recommend using whatever version of Magic Forms you most recently had working. New installs on older version of October won't be supported.

Thanks!
